server/test/role.spec.js

```diff
@@ -0,0 +1,225 @@
+import supertest from 'supertest';
+import should from 'should';
+import app from '../../server';
+import testData from './helpers/specHelper';
+
+const server = supertest.agent(app);
+
+let adminToken, regularUserToken, newRoleId, newRoleTitle;
+
+describe('Roles:', () => {
+  before((done) => {
+    server.post('/users')
+    .send(testData.adminUserRole)
+    .end((error, res) => {
+      adminToken = res.body.token;
+      server.post('/users')
+      .send(testData.regularUserRole)
+      .end((error1, res) => {
+        regularUserToken = res.body.token;
+        done();
+      });
+    });
+  });
```

Comment:
The above block does not seem to be running any test, rather it is providing a `prerequisite data` for the following tests. You may want to rename it appropriately